express/parsers/utils.py

```diff
@@ -16,3 +16,47 @@ def find_file(name, path):
         for file in files:
             if name in file:
                 return os.path.join(root, file)
+
+
+def find_files(name, path):
+    matches = []
+    for root, dirs, files in os.walk(path, followlinks=True):
+        for file_ in files:
+            if name in file_:
+                matches.append(os.path.join(root, file_))
+    return matches
+
+
+def get_element_counts(basis):
+    """
+    Returns chemical elements with their count wrt their original order in the basis.
+    Note: entries for the same element separated by another element are considered separately.
+    [{"count":1, "value":"Zr"}, {"count":23, "value":"H"}, {"count":11, "value":"Zr"}, {"count":1, "value":"H"}]
+    """
+    element_counts = []
+    previous_element = None
+    for index, element in enumerate(basis["elements"]):
+        if previous_element and previous_element["value"] == element["value"]:
+            element_counts[-1]["count"] += 1
+        else:
+            element_counts.append({
+                "count": 1,
+                "value": element["value"]
+            })
+        previous_element = basis["elements"][index]
+    return element_counts
+
+
+def to_poscar(lattice, basis):
```

Comment:
1. lattic_basis_to_poscar
2. either append "_cartesian" to the function name of add a parameter with units to handle crystal coordinates too

Reply:
Done.